x-pack/plugins/observability/public/pages/slos_page/helpers/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ConfigSchema } from '../../../plugin';
+
+function isSloFeatureEnabled(config: ConfigSchema): boolean {
```

Comment:
Usually, I saw the helper's file name the same as the helper's name since the file name `index.ts` is not showing what is inside of it.
I wish we had the discussion about the structure sooner :(

Reply:
Yes I agree, going to extract into its own file, and export from the index (for grouped import) 👍🏻